cli/commands/benchmark-url.mjs

```diff
@@ -133,6 +133,7 @@ function benchmarkURL( params ) {
 	const metrics = {};
```

Comment:
See my previous reply, maybe add a comment somewhere on top of this file linking to a Gist with the GitHub workflow?

Reply:
Yep, sounds good to me. Added.